### PR TITLE
Update April-1 preview with changes from March-1 preview

### PR DESCRIPTION
**Pull request by abhahn**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/analyzeconversations.json
Comment (heaths) [reply specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/analyzeconversations.json:1]: No changes detected.

File: specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/common.json
Comment (heaths) [reply specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/common.json:16]: No changes detected.

File: specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json
Comment (heaths) [reply specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json:189]: No changes detected.

**Review by heaths — APPROVED** [review]
Same comments as the other PR to merge 2022-03-01-preview changes into a newer one. Should I expect to see some additions or something, though? But if this is right, I'm certainly not going to block it. Just surprised is all.

File: specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/examples/conversations/Orchestrator_direct_target.json
Comment (abhahn) [reply specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/examples/conversations/Orchestrator_direct_target.json]: Yeah, it looks to be incorrect in the main branch, so I just inherited the error when I pulled everything over.

https://github.com/Azure/azure-rest-api-specs/blob/24c1d13f898c3a909a352d33e351c9b4c2da1195/specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/examples/conversations/Orchestrator_direct_target.json#L6

I did a find/replace on 2022-03-01-preview, so that's why I missed it.

I just updated it to the correct version here.

**Review by heaths — APPROVED** [review]

**Comment by heaths** [comment]
@JeffreyRichter or @johanste can we get a breaking change exemption. This is the same as 2022-03-01-preview where the `error` property of the error object was made required. Honestly not sure why we're still seeing it, though. Perhaps after we can GA a newer version of common.json (should for 2022-05-01) it'll go away.

File: specification/cognitiveservices/data-plane/Language/readme.md
Comment (heaths) [inline specification/cognitiveservices/data-plane/Language/readme.md:14]: This should be updated for the April release i.e. `release_2022-04-01_preview` and then add a block for that as well, basically copying lines 23 through 35 (in the right/new file) below.

Each package/release shoul dhave it's own yaml section, but the top-level `tag:` and friends are updated for each subsequent release. Before they were duplicated, and it wasn't clear from diffs until I merged March's readme.md.

**Review by heaths — CHANGES_REQUESTED** [review]
Just one important change and this is good to merge, though I'll probably still need an override because of expected validation failures.

File: specification/cognitiveservices/data-plane/Language/readme.md
Comment (heaths) [reply specification/cognitiveservices/data-plane/Language/readme.md:14]: Also, since you're removing analyzeconversations.json, make sure the release_2022-04-01-preview section doesn't reference it in `input-file:`.

**Review by heaths — APPROVED** [review]

**Comment by heaths** [comment]
@ShivangiReja @AlexanderSher any idea why autorest is saying no input files were passed despite the tooling finding modified files?

https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1493551&view=logs&j=2be18a23-047c-5c9f-61d7-60a33437d1f1&t=90227b5e-3523-5803-3e16-3788cf693a16&l=62

**Comment by heaths** [comment]
@akning-ms @lirenhe talking with @ShivangiReja offline, it sounds like this might be a tooling issue instead. The error isn't very descriptive so I'm not really sure what to make of it.

**Comment by akning-ms** [comment]
It looks readme file issue. + @dw511214992 to help and confirm

**Comment by heaths** [comment]
@johanste @mayurid @lmazuel do you see any reason to hold this PR up for the SDK issue? We could fix that in parallel.

**Pull request merged**
